uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/apis/work-pattern-api_test.js

```diff
@@ -103,7 +105,7 @@ define([
       });
 
       describe('when multiple contact ids are passed', function () {
-        var multipleContacts = [dummyContactId, dummyContactId + 1];
+        var multipleContacts = [contactId, contactId + 1];
```

Comment:
If we a going to change contactId to string, then this should be 
`var multipleContacts = [contactId, (parseInt(contactId) + 1).toString()];`